ranks.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from semesters import get_current_semester, semester_date
+import models
```

Comment:
I am getting a 500 error in this file, and I am _pretty_ sure it is because of this line.  I think you need to explicitly do `from qwer import Uiop, Asdf, Ghjk‘.

Reply:
I am also getting that. the `from x import y, z, a` is also causing problems. Its the difference between an `ImportError: Cannot import...` and a `NameError: global name ... not defined`. I am not 100% sure how to resolve the issue, still working it. 

I think the root of the problem is ranks requires some classes from models, which also needs to import ranks to run.